ibis/expr/api.py

```diff
@@ -1783,7 +1800,8 @@ def _timestamp_strftime(arg, format_str):
     minute=_extract_field('minute', _ops.ExtractMinute),
     second=_extract_field('second', _ops.ExtractSecond),
     millisecond=_extract_field('millisecond', _ops.ExtractMillisecond),
-    truncate=_timestamp_truncate
+    truncate=_timestamp_truncate,
+    time=_timestamp_time,
```

Comment:
Since it's a type in many databases. Here's postgres:

```
ibis_testing=# select cast(timestamp_col as time) as "timestamp_col::time", timestamp_col from functional_alltypes limit 5;
 timestamp_col::time |     timestamp_col     
---------------------+-----------------------
 00:00:00            | 2010-06-01 00:00:00
 00:01:00            | 2010-06-01 00:01:00
 00:02:00.1          | 2010-06-01 00:02:00.1
 00:03:00.3          | 2010-06-01 00:03:00.3
 00:04:00.6          | 2010-06-01 00:04:00.6
(5 rows)
```